prow/cluster/jobs/istio/operator/istio.operator.master.yaml

```diff
@@ -22,28 +22,32 @@ presubmits:
   istio/operator:
 
   - name: operator-lint-master
-    <<: *job_template
-    context: prow/lint.sh
+    path_alias: istio.io/operator
+    decorate: true
     always_run: true
     spec:
       containers:
-      - <<: *istio_container
+      - image: golang:1.12.7
         command:
-        - entrypoint
-        - prow/lint.sh
+        - make
+        - lint
```

Comment:
Does this test need the lint tool preinstallled in the image?

Reply:
`make lint` runs `scripts/run_golangci.sh` which downloads the linter binary each time. The previous prow based test worked the same way.